### ISSUE TITLE
Programmatic usage produces empty output

### ISSUE THREAD

**Issue by NickHeiner** [wessberg/cjstoesm#18]
I'm using this script, which I adapted from the README:

```js
const {transform} = require('cjstoesm');

async function main() {
    const input = require.resolve('./fixtures');

    console.log('Running on', input);
    const result = await transform({
        input
    });

    console.log(result);
}

main();

```

The output is:

```
$ node tools/codemod/cjs-to-esm/codemod.js                                                                                                                                                                                                                                                                                                                          
Running on /Users/nheiner/code/tvui/tools/codemod/cjs-to-esm/fixtures/index.js
{ files: [] }
```

**Comment by wessberg** [comment]
Hi there,

Thanks for making me aware of this potential regression.
Can you provide some more details, such as:

- What's your OS/subsystem?
- What's the file extensions of your source files?
- Or even better, can you provide a minimal repro that causes this behavior?

I will set up some test cases and see if U can reproduce this easily myself, but any details you might have will aid me.

**Comment by wessberg** [comment]
Hi again,

I was able to reproduce and fix this problem easily. A fix has been released in [v1.1.0](https://github.com/wessberg/cjstoesm/releases/tag/v1.1.0)

**Comment by ruanyl** [comment]
It seems it's reproducible on v1.1.4 again, it's able to produce output with cli but not with API use. I'm on OSX
```typescript
const result = await transform({
  input: args.path,
  outDir: 'dist',
  write: false,
  debug: true,
})

console.log('result: ', result.files)
```


Output:
```
[DEBUG] Options: {
  input: [
    '/project/reapex-cli/node_modules/react-dom/cjs/react-dom.development.js'
  ],
  outDir: '/project/reapex-cli/examples/counter/dist',
  cwd: '/project/reapex-cli/examples/counter',
  write: false,
  debug: true,
  preserveModuleSpecifiers: 'external'
}
[DEBUG] Matched files: "/Users/yulongruan/project/reapex-cli/node_modules/react-dom/cjs/react-dom.development.js"
result:  []
```
